Resources/Textures/Floof/Objects/HypnoTempPlushies/staffer_plushies.rsi/meta.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "license": "CC BY-NC-SA 4.0",
+    "copyright": "JTCircus",
```

Comment:
I suggest providing more information, like discord ID, github account, ect... this is fine too